layers/meta-resin-qemu/conf/samples/local.conf.sample

```diff
@@ -66,3 +66,5 @@ PACKAGECONFIG_append_pn-qemu-native = " sdl"
 PACKAGECONFIG_append_pn-nativesdk-qemu = " sdl"
 
 CONF_VERSION = "1"
+
+HOSTTOOLS += "docker iptables"
```

Comment:
Why iptables?

Reply:
iptables is a runtime dependency of dockerd